src/renderer/pages/player-page.js

```diff
@@ -585,6 +586,8 @@ function renderPlayerControls (state) {
 // Renders the loading bar. Shows which parts of the torrent are loaded, which
 // can be 'spongey' / non-contiguous
 function renderLoadingBar (state) {
+  if (config.IS_TEST) return // Don't integration test the loading bar. Screenshots won't match.
```

Comment:
IMO we want to use `config.IS_TEST` as little as possible.

In this case, it saves us from having to make transparent screenshots for every shot of the player page. Feels better b/c regular screenshots are easier to update when the UI changes